### DISCUSSION TITLE
Is there a way to save the best model?

### DISCUSSION THREAD

**Discussion by keiohta** [nerfstudio-project/nerfstudio#2136] [Q&A]
Hi,

Currently I save only the latest checkpoint by specifying `save_only_latest_checkpoint = True` in the `TrainerConfig` but I found the model overfits to the data from the validation loss. So, I want to save only the best model that achieves the lowest validation loss. Is there a way to do this? Thanks!
